roles/imageset_mirroring/README.md

```diff
@@ -0,0 +1,186 @@
+# imageset_mirroring
+
+Mirror OpenShift operator catalogs and images using oc-mirror.
+
+## Description
+
+This role provides three different approaches to mirror operators from a source catalog to a target registry:
+
+- **mirror-latest**: Mirror only the latest version of specified operators
+- **mirror-all**: Mirror all channels and versions of specified operators
+- **mirror-custom**: Mirror using a pre-defined custom ImageSetConfiguration file
+
+## Requirements
+
+- `oc-mirror` binary (automatically downloaded by the role)
+- Network access to source and target registries
+- Valid authentication for registries (if required)
+
+## Role Variables
+
+### Common Variables
+
+| Variable | Type | Required | Default | Description |
+|----------|------|----------|---------|-------------|
+| `im_mirror_target` | string | yes | - | Target registry for mirroring (e.g., `registry.example.com:5000`) |
```

Comment:
I think the use of  `mirror` in vars could be avoided since `im` already stands for imageset_mirroring.

For `im_mirror_target` it could be something like `im_target` or `im_target_registry` or something like that.